ui/analyse/src/study/relay/relayTeams.ts

```diff
@@ -70,16 +70,17 @@ const renderTeams = (
   cloudEval?: MultiCloudEval,
 ): MaybeVNodes =>
   teams.table.map(row => {
-    const firstTeam = row.teams[0];
+    const firstTeam = row.teams[1];
+    const secondTeam = row.teams[0];
```

Comment:
This assumes that the second team is first, and the first team is second.
I doubt that it's always true, but if it was, then it should be fixed upstream in the backend, and not reverted in the frontend.

Reply:
Well... I may have done it in the frontend, but this is currently how lila is generating the teams (not just in this case at all)

Reply:
The first random team broadcast I looked at has the first team's first player with the white pieces:
https://lichess.org/broadcast/french-cup-grand-finals-chartres-2024--womens-top-12-club-championship/semi-finals/EisjcYTg#teams

Reply:
Oh...